Libplanet/Blockchain/BlockChain.cs

```diff
@@ -174,13 +174,16 @@ IEnumerable<IRenderer<T>> renderers
 
             if (Count == 0)
             {
-                Append(
-                    genesisBlock,
-                    currentTime: genesisBlock.Timestamp,
-                    renderBlocks: !inFork,
-                    renderActions: !inFork,
-                    evaluateActions: !inFork
-                );
+                if (!inFork)
```

Comment:
How about merging two conditions into one using an `&&` operator?

Reply:
Oh, [fair enough](https://github.com/planetarium/libplanet/pull/1063#discussion_r514946332).

Reply:
It will throw exception in next else if statement 😢